src/uct/ib/base/ib_md.c

```diff
@@ -558,8 +558,8 @@ ucs_status_t uct_ib_mem_prefetch(uct_ib_md_t *md, uct_ib_mem_t *ib_memh,
     } while ((ret == EAGAIN) && (retry++ < md->config.reg_retry_cnt));
 
     if (ret) {
-        ucs_error("ibv_advise_mr(addr=%p length=%zu key=%x) returned %d: %m",
-                  addr, length, ib_memh->lkey, ret);
+        ucs_diag("ibv_advise_mr(addr=%p length=%zu key=%x) returned %d: %m",
```

Comment:
IMO in error case it's not interesting since in case of err EAGAIN it always will be reg_retry_cnt. I would rather add some debug/trace with retries in good flow.